config.toml.example

```diff
@@ -158,11 +158,9 @@
 
 # Python interpreter to use for various tasks throughout the build, notably
 # rustdoc tests, the lldb python interpreter, and some dist bits and pieces.
-# Note that Python 2 is currently required.
+# Note that Python 3 is currently required.
 #
-# Defaults to python2.7, then python2. If neither executable can be found, then
-# it defaults to the Python interpreter used to execute x.py.
-#python = "python2.7"
+#python = "python3"
```

Comment:
I would personally prefer that this is "python".

Reply:
Many of the builders are missing a `python` executable. See the first failure in https://github.com/rust-lang/rust/pull/66396 where i hit this issue.

Reply:
It can be solved by installing `python` package which provides symlink.